landscape/client/manager/usgmanager.py

```diff
@@ -202,7 +202,7 @@ def _set_reboot_required(self) -> None:
     def _spawn_usg(
         self,
         action: str,
-        profile: str,
+        profile: str | None,
```

Comment:
Just a heads up: this should be okay unless we need to port to focal, in which case we'll need to use `Optional`.